Fixed Python Directories and Added testing to App

jkaufy opened the pull request:
- added setup.py, readme, license, manifest
- fixed file structure

jkaufy committed:
Fixed Python Directories

- added setup.py, readme, license, manifest
- fixed file structure

jkaufy committed:
Fix file structure for PIP

kaitlyndlee commented on app/.babelrc:
```diff
@@ -1,5 +1,13 @@
 {
-  "presets": ["airbnb"],
+  "presets": [
```
What does this do? I wouldn't think something in the app folder would affect the jupyter notebook stuff.

kaitlyndlee commented:
Do we need the egg and build stuff? We shouldn't have any build stuff in the source code.

jkaufy committed:
Deleting files for PIP packaging, Adding testing for AstroMath

- deleted dist, build
- Fixed AstroMath
- Added dev dependencies to package.json
- Added test script to NPM
- Added test folder with astroMath_test.js
- fixed AstroMath and MousePosition

kaitlyndlee commented on app/test/AstroMath_test.js:
```diff
@@ -0,0 +1,86 @@
+  it('Testing toRadians: 360 degrees should be 6.283185307179586', function() {
+    // add an assertion
+    let majorRadius = testMath.toRadians(360);
+    expect(majorRadius).to.equal(6.283185307179586);
```
Is there a way we can expect the answer to be near instead of having all of the decimal places?

jkaufy committed:
Fixed testing Floating Point issue

kaitlyndlee commented on app/test/AstroMath_test.js:
```diff
@@ -0,0 +1,86 @@
+  it('Testing toRadians: 360 degrees should be 6.28', function() {
+    // add an assertion
+    let majorRadius = testMath.toRadians(360);
+    expect(majorRadius).to.be.closeTo(6.28, 0.01);
```
Sorry this was my mistake. Could we actually compare the values to the radian value with pi in it? Like 360 == 2pi?

jkaufy replied:
Fixed

kaitlyndlee commented on app/test/AstroMath_test.js:
```diff
@@ -0,0 +1,86 @@
+    })
+
+    // testing lonTo360 function
+    it('Testing lonTo360 Non-Cylindrical: -180 should be 180 ', function() {
```
I could be reading this wrong, but doesn't -180 become 0 in the 0-360 domain?

jkaufy replied:
Our code was wrong I fixed the test

jkaufy committed:
Change tests for radians to be in values of pi

jkaufy committed:
Fixed longitude to 360 code

The longtitude to 360 for polar projections was wrong

jkaufy committed:
updated test and fixed lonTo360 latitude code

jkaufy committed:
Updated Mouse Position

Fixed issue where it would show coordinates when the user's mouse wasn't on the map

scottaames committed:
Update MousePosition.js

Don't need to import as L, it is defined in the configuration files that L is for leaflet
